keylime/ima/ima_dm.py

```diff
@@ -367,6 +366,22 @@ def invalid(self) -> Failure:
         return failure
 
 
+def _strtobool(val: str) -> bool:
```

Comment:
maintain the original name now that it is not imported?

Reply:
No because it is only used internally of this module.

Reply:
agreed

Reply:
So `_strtobool` is fine right?

Reply:
yes, makes sense